src/sql/azdata.d.ts

```diff
@@ -3222,6 +3222,9 @@ declare module 'azdata' {
 		headerCssClass?: string;
 		toolTip?: string;
 		type?: ColumnType;
+		/**
+		 * @deprecated options property is deprecated, use specific column types to access the options directly
+		 */
 		options?: CheckboxColumnOption | TextColumnOption;
```

Comment:
I am not able to update the options to include new types without updating azdata.d.ts directly. using more specific column types can solve this problem.